dev-python/nose/nose-1.3.7_p20211111_p1-r1.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( python3_{8..10} pypy3 )
+PYTHON_COMPAT=( python3_{8..11} pypy3 )
```

Comment:
Nose is a package that we don't want to port to python3_11, it doesn't work with python3_11 and is not actively maintained upstream any more. Instead ebuilds that depend on it to run the tests should be ported to pytest, often this is an easy change: `distutils_enable_tests nose` --> `distutils_enable_tests pytest`. Sometimes it is more complicated though.

Reply:
As the current maintainer of nose fork we use at gentoo, I can say that nose **is not** ready for python 3.11!

Reply:
Thanks, nose is dev-python/astroid's test bdepend, I will try to use pytest instead of nose.